### PR TITLE
Changing sdk package name to cosmos for user agent

### PR DESCRIPTION
**Pull request by simplynaveen20**
After this user agent will change from

azsdk-java-**cosmosdb-java-sdk**/4.4.0-beta.1 Windows10/10.0 JRE/11.0.6 test-application-id

to 

azsdk-java-**cosmos**/4.4.0-beta.1 Windows10/10.0 JRE/11.0.6 test-application-id

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java [hunk 1]
```diff
@@ -261,7 +261,7 @@
         public static final String AZURE_COSMOS_PROPERTIES_FILE_NAME = "azure-cosmos.properties";
 
         public static final String SDK_VERSION = CoreUtils.getProperties(AZURE_COSMOS_PROPERTIES_FILE_NAME).get("version");
-        public static final String SDK_NAME = "cosmosdb-java-sdk";
+        public static final String SDK_NAME = "cosmos";
```
Comment (kirankumarkolli) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java:264]: Isn't there a test to verify the prefix?

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java [hunk 2]
```diff
@@ -87,7 +87,7 @@ private String getUserAgentFixedPart() {
         osName = osName.replaceAll("\\s", "");
         String geteUserAgentFixedPart =
             UserAgentContainer.AZSDK_USERAGENT_PREFIX +
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java:89]: please also replace `UserAgentContainer.AZSDK_USERAGENT_PREFIX` with hard coded value for the test

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java [hunk 3]
```diff
@@ -87,7 +87,7 @@ private String getUserAgentFixedPart() {
...
-                HttpConstants.Versions.SDK_NAME +
+                "cosmos" +
                 "/" +
                 HttpConstants.Versions.SDK_VERSION +
                 SPACE +
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java:93]: ditto, for the space, the only constant we should use here is the `HttpConstants.Versions.SDK_VERSION`

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java [hunk 4]
```diff
@@ -87,7 +87,7 @@ private String getUserAgentFixedPart() {
         osName = osName.replaceAll("\\s", "");
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java:87]: btw, this class, has a test named `UserAgentIntegrationTest()` (from prior useragent PR), please change that to
`userAgentIntegration()` to follow test name code style.

**Pull request merged**